static/src/javascripts-legacy/projects/common/modules/experiments/tests/epic-for-brexit-cohort.js

```diff
@@ -0,0 +1,82 @@
+define([
+    'lodash/utilities/template',
+    'commercial/modules/user-features',
+    'common/modules/commercial/contributions-utilities',
+    'lib/config',
+    'raw-loader!common/views/epic-for-brexit-cohort.html',
+    'svgs/icon/quote.svg',
+    'common/modules/commercial/acquisitions-epic-testimonial-parameters'
+], function (
+    template,
+    userFeatures,
+    contributionsUtilities,
+    config,
+    epicForBrexitCohortTemplate,
+    quoteSvg,
+    acquisitionsEpicTestimonialParameters
+) {
+
+    function hasBrexitTag(){
```

Comment:
you might want to use the targeting tool instead of hardcoding this - give me a shout if you want to know how